src/PharoThings-Hardware-Core.package/PotBoard.class/instance/addExistingConnector.with..st

```diff
@@ -0,0 +1,8 @@
+initialization
+addExistingConnector: aString with: pinsArray
+	| connector pinsArrayExisting |
+	pinsArrayExisting  := (connectors at: 1) pins.
```

Comment:
And "connectors at: 1" is also wrong. It should be "self connectorNamed: aString".

Also I would rename method to #extendConnector:with:. And ask existing connector to #addPins: instead of performing this concatination here.

Reply:
I changed to "self connectorNamed: aString" and renamed the method. 
I ask the existing connector, but the pins are an array, and I can not add more pins to this array. 
Because this I remove it and create another array, concatenating the older and the new one. 
I didn't find this method #addPins.

Reply:
Yes, you need to add #addPins: method to connector class. This way we will improve board model and API

Reply:
Done